frontend/src/pages/RecoverPage.jsx

```
@@ -1,9 +1,279 @@
+  const handleSendOtp = async (e) => {
+    e.preventDefault();
+    if (!email) {
+      setErrorMessage("Email is required");
+      setShowError(true);
+      setTimeout(() => setShowError(false), 3000);
+      return;
+    }
+
+    const result = await sendResetOtp(email);
+
+    if (result && result.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)